index.html

```diff
@@ -185,6 +186,42 @@ <h1 class="multi-post-header">Multi-Post Stories</h1>
                 </div>
             </section>
         </section>
+        <form action="https://formspree.io/f/xjvlzgoa" method="post">
+            <h2 class="form-header">CONTACT US</h2>
+            <P class="form-paragraph">If you have an application you are interested
+                in developing, a feature that you need built or
+                a project that needs coding. I&#x27;d love to help with it.
+            </P>
+            <ul>
+                <li>
+                    <label for="name"></label>
+                    <input type="text" name="user_name" id="name" minlength="2" maxlength="30" required placeholder="Text Input">
+                </li>
+                <li>
+                    <label for="email"></label>
+                    <input type="email" name="user_email" id="email" placeholder="jeremy@gmail.com" required>
+                </li>
+                <li>
+                    <label for="msg"></label>
+                    <textarea name="user_message" id="msg" placeholder="Enter text here" required maxlength="500"></textarea>
+                </li>
+            </ul> 
+            <button type="submit" class="see-project-form">See project</button>
+        </form>
+        <footer>
+            <!-- <hr class="hr-footer"> -->
+            <ul class="footer-account-links">
+                <li><a href="https://github.com/aron-helu" class="link-footer"><img src="./image/Vector.svg" alt="github-icon"></a></li>
+                <li><a href="https://www.linkedin.com/in/aron-abraham-90a4321b0/" class="link-footer"><img src="./image/Icon-Linkedin.svg" alt="Linkedin-github"></a></li>
+                <li><a href="#" class="link-footer"><img src="./image/Vector_hand.svg" alt="Peace-hand"></a></li>
+                <li><a href="#" class="link-footer"><img src="./image/Vector_Twit.svg" alt="Twitter-icon"></a></li>
+                <li><a href="#" class="link-footer"><img src="./image/Vector_M.svg" alt="Medea"></a></li>
+            </ul>
+            <h2>aronmilahela@gmail.com</h2>
```

Comment:
- [ ] Kindly make sure the social icons and the email link are correctly centered like from the Figma template(you can use flexbox to easily center them) and the `bottom` **horizontal** line to match the styling from the Figma. Also, make sure you remove the comments code lines to make your code clean. By fixing these issues your project will stick to the Figma template.
#### Figma design
![ff](https://user-images.githubusercontent.com/83042742/172514207-7b32d7a5-9e89-4979-b2f6-24f5d27dabe8.png)
#### Your design
![fs](https://user-images.githubusercontent.com/83042742/172514282-76959313-7c7c-404b-a0d3-0851f1d30d5e.png)